### ISSUE TITLE
[Interactive] Lightbulb menu for adding a using directive crashes VS

### ISSUE THREAD

**Issue by Suchiman** [dotnet/roslyn#27466]
**Version Used**: 15.8 Preview 2

**Steps to Reproduce**:

1. Open C# Interactive
2. Paste `DateTime.Parse("", CultureInfo.CurrentCulture);`
3. Press `CTRL + .` to invoke Lightbulb menu

**Expected Behavior**:
Lightbulb menu opens, shows preview and allows me to apply codefix.

**Actual Behavior**:
Lightbulb menu opens but VS crashes before showing the preview with following stacktrace:
```
Exception: System.NullReferenceException
   at Microsoft.VisualStudio.LanguageServices.Telemetry.ProjectTypeLookupService.GetProjectType(Microsoft.CodeAnalysis.Workspace, Microsoft.CodeAnalysis.ProjectId)
   at Microsoft.CodeAnalysis.Editor.Shared.Extensions.IWorkspaceExtensions.GetLanguageAndProjectType(Microsoft.CodeAnalysis.Workspace, Microsoft.CodeAnalysis.ProjectId, System.String ByRef, System.String ByRef)
   at Microsoft.CodeAnalysis.Editor.Implementation.Suggestions.SuggestedActionWithNestedFlavors+<GetPreviewAsync>d__11.MoveNext()
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(System.Threading.Tasks.Task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(System.Threading.Tasks.Task)
   at Microsoft.VisualStudio.Language.Intellisense.Implementation.LightBulbSession+<GetPreviewAsync>d__41.MoveNext()
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(System.Threading.Tasks.Task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(System.Threading.Tasks.Task)
   at Microsoft.VisualStudio.Language.Intellisense.Implementation.LightBulbMenuItem+<GetOrCreatePreviewAsync>d__43.MoveNext()
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(System.Threading.Tasks.Task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(System.Threading.Tasks.Task)
   at Microsoft.VisualStudio.Language.Intellisense.Implementation.LightBulbMenuItem+<InitializeFlavoredAsync>d__19.MoveNext()
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(System.Threading.Tasks.Task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(System.Threading.Tasks.Task)
   at Microsoft.VisualStudio.Language.Intellisense.Implementation.LightBulbMenuItem+<InitializeAsync>d__17.MoveNext()
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(System.Threading.Tasks.Task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(System.Threading.Tasks.Task)
   at Microsoft.VisualStudio.Language.Intellisense.Implementation.LightBulb+<OnMenuItemGotKeyboardFocus>d__59.MoveNext()
   at System.Runtime.CompilerServices.AsyncMethodBuilderCore+<>c.<ThrowAsync>b__6_0(System.Object)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(System.Delegate, System.Object, Int32)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(System.Object, System.Delegate, System.Object, Int32, System.Delegate)
   at System.Windows.Threading.DispatcherOperation.InvokeImpl()
   at System.Windows.Threading.DispatcherOperation.InvokeInSecurityContext(System.Object)
   at MS.Internal.CulturePreservingExecutionContext.CallbackWrapper(System.Object)
   at System.Threading.ExecutionContext.RunInternal(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object, Boolean)
   at System.Threading.ExecutionContext.Run(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object, Boolean)
   at System.Threading.ExecutionContext.Run(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object)
   at MS.Internal.CulturePreservingExecutionContext.Run(MS.Internal.CulturePreservingExecutionContext, System.Threading.ContextCallback, System.Object)
   at System.Windows.Threading.DispatcherOperation.Invoke()
   at System.Windows.Threading.Dispatcher.ProcessQueue()
   at System.Windows.Threading.Dispatcher.WndProcHook(IntPtr, Int32, IntPtr, IntPtr, Boolean ByRef)
   at MS.Win32.HwndWrapper.WndProc(IntPtr, Int32, IntPtr, IntPtr, Boolean ByRef)
   at MS.Win32.HwndSubclass.DispatcherCallbackOperation(System.Object)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(System.Delegate, System.Object, Int32)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(System.Object, System.Delegate, System.Object, Int32, System.Delegate)
   at System.Windows.Threading.Dispatcher.LegacyInvokeImpl(System.Windows.Threading.DispatcherPriority, System.TimeSpan, System.Delegate, System.Object, Int32)
   at MS.Win32.HwndSubclass.SubclassWndProc(IntPtr, Int32, IntPtr, IntPtr)
```

**Comment by Neme12** [comment]
Was this fixed in #27127? Tagging @sharwell

**Comment by sharwell** [comment]
@Neme12 This just came up internally too. I think you're right.
